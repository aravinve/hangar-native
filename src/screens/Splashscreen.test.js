import { render } from "@testing-library/react-native";
import React from "react";
import SplashScreen from "./SplashscreenScreen";

beforeEach(() => {
  jest.useFakeTimers();
});

afterAll(() => {
  jest.resetAllMocks();
  jest.resetModules();
});

describe("<InitialScreen />", () => {
  it("should render and match InitialScreen snapshot", async () => {
    const renderedSplashScreen = await render(<SplashScreen />);
    expect(renderedSplashScreen).toMatchSnapshot();
  });
});
