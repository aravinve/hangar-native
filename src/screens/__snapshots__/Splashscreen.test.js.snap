// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InitialScreen /> should render and match InitialScreen snapshot 1`] = `
<View
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <RCTSafeAreaView
    emulateUnlessSupported={true}
    style={
      Object {
        "display": "flex",
        "height": "100%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <Image
      source={
        Object {
          "testUri": "../../../src/assets/logo.png",
        }
      }
      style={
        Object {
          "alignContent": "center",
          "alignSelf": "center",
          "height": 150,
          "width": 150,
        }
      }
      testID="logoImage"
    />
    <Text
      style={
        Object {
          "color": "#000000",
          "fontSize": 32,
          "marginTop": 20,
          "textAlign": "center",
        }
      }
      testID="logoLabel"
    >
      Hangar
    </Text>
    <View />
  </RCTSafeAreaView>
</View>
`;
